src/host/polyfills/simpleView.ts

```diff
@@ -1,10 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import ToolsHost from "../toolsHost";

```

Comment:
Is there a way to fail te build if some of this regex do not match? Otherwise it will break some features without people noticing if the source code change

Reply:
There probably is, but it requires the build to pull down and build the opensource version of Edge in order to test against the elements code. This is a bunch of work to do for the CI system. I can open an issue for it though.

Reply:
Added #73